metashare/api/tests/jobs.py

```diff
@@ -536,25 +537,35 @@ class TestSubmitReview:
     def test_good(self, task_factory, user_factory):
         with patch("metashare.api.jobs.get_repo_info") as get_repo_info:
             user = user_factory()
-            task = task_factory(pr_is_open=True, commits=[{"id": "none"}])
+            task = task_factory(
+                pr_is_open=True, review_valid=True, review_sha="test_sha"
+            )
             task.finalize_submit_review = MagicMock()
             pr = MagicMock()
             repository = MagicMock(**{"pull_request.return_value": pr})
             get_repo_info.return_value = repository
             submit_review(
                 user=user,
                 task=task,
-                data={"notes": "Notes", "status": "APPROVE", "delete_org": None},
+                data=OrderedDict(
```

Comment:
I don't think there's any reason for this to be an `OrderedDict`; we don't rely on ordering anywhere.

Reply:
Sure -- I used it because that seems to be what `serializer.validated_data` is, and I was trying to make the test match reality as much as possible.

Reply:
`OrderedDict` is entirely the same as `dict` except that when you iterate over keys, you are guaranteed to get them in insertion order (which is gonna be the case for `dicts` in Py 3.8 anyway, I believe?), so while this is more accurately like the reality, it's not to much benefit, since we don't iterate over the keys ever.